ci-operator/config/openshift/oc-mirror/openshift-oc-mirror-release-4.11.yaml

```diff
@@ -10,6 +10,10 @@ resources:
       memory: 200Mi
 test_binary_build_commands: make build
 tests:
+- as: sanity
+  commands: make tidy && git diff --exit-code
+  container:
+    from: src
```

Comment:
why no images job and promotion for 4.11?

Reply:
Added this. Thanks for reviewing!